tests/unit/test_download.py

```diff
@@ -413,6 +413,41 @@ def test_unpack_file_url_thats_a_dir(self, tmpdir, data):
         assert os.path.isdir(os.path.join(self.build_dir, 'fspkg'))
 
 
+@pytest.mark.parametrize('exclude_dir', [
+    '.nox',
+    '.tox'
+])
+def test_unpack_file_url_excludes_expected_dirs(tmpdir, exclude_dir):
+    src_dir = tmpdir / 'src'
+    dst_dir = tmpdir / 'dst'
+    src_included_file = Path.joinpath(src_dir, 'file.txt')
```

Comment:
Sorry, my previous comment was not very good - I meant using the `joinpath` method on `src_dir` which is now a `Path`, like `src_dir.joinpath('file.txt')`. Likewise with `joinpath` and `touch` below.